inst/rmarkdown/templates/template-name/skeleton/skeleton.Rmd

```diff
@@ -1,21 +1,41 @@
 ---
-title: "The post title"
-date: "YYYY-MM-DDTHH:MM:SS"
+params:
+  post_title: "The Post Title"
+title: "`r params$post_title`"
+date: "`r format(Sys.time(), '%Y-%m-%dT%H:%M:%S')`"
 output: hugodown::md_document
+slug: "`r gsub(' ', '-', tolower(params$post_title))`"
 status: "publish"
-slug: "the-post-slug"
+comment_status: "open"
```

Comment:
I am a bit undecided, do you think most people prefer to have comments open?

Reply:
that's the default setup on WordPress.